third_party/fiat/p384_64.h

```diff
@@ -18,14 +18,9 @@
 #include <stdint.h>
 typedef unsigned char fiat_p384_uint1;
 typedef signed char fiat_p384_int1;
-#ifdef __GNUC__
-#  define FIAT_P384_FIAT_EXTENSION __extension__
-#else
-#  define FIAT_P384_FIAT_EXTENSION
-#endif
-
-FIAT_P384_FIAT_EXTENSION typedef signed __int128 fiat_p384_int128;
-FIAT_P384_FIAT_EXTENSION typedef unsigned __int128 fiat_p384_uint128;
+// Pedantic warnings can be disabled by adding prefix __extension__.
```

Comment:
This is the p256_64.h way https://github.com/awslabs/aws-lc/blob/main/third_party/fiat/p256_64.h#L18

So, we actually don't care about not hitting a compiler that doesn't define `__GNUC__`?